agent/stats/engine_test.go

```diff
@@ -54,6 +54,10 @@ func (engine *MockTaskEngine) ListTasks() ([]*api.Task, error) {
 	return nil, nil
 }
 
+func (engine *MockTaskEngine) GetTaskByArn(arn string) (*api.Task, bool) {
```

Comment:
Yeah, just following the [map lookup semantics](https://blog.golang.org/go-maps-in-action) in golang.